src/sensor/SemanticLidarSensor.py

```diff
@@ -106,12 +107,16 @@ def compute_detected_objects(self):
         # https://github.com/usdot-fhwa-stol/carma-utils/issues/194
         detected_objects = self.apply_occlusion(detected_objects, actor_angular_extents, hitpoints,
                                                detection_thresholds)
+        #compare ground truth for position and rotation for SUMO vehicle
+        self.store_groundtruth_pose_angular_diff(detected_objects)
```

Comment:
Why did we need to separate and store another function like this? 
Couldn't we have just moved the `update_velocity_angularVelocity` to here?

Reply:
The part where it actually calculates the timestamp part, which is assigned after the add_noise function, so I separate those two parts

Reply:
okay that makes sense